TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -190,6 +191,15 @@ public class TwilioVerifyBuilder {
     return self
   }
 
+  /// Set the attrAccessible parameter that will be used for Keychain storage.
+  /// - Parameter attrAccessible: Used to specify the security protection of Keychain items [kSecAttrAccessible](https://developer.apple.com/documentation/security/ksecattraccessible)
+  /// The default option is [kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessibleafterfirstunlockthisdeviceonly)
+  /// which means that the Keychain item can only be accessed after the device is unlocked for the first time on the current device and cannot be migrated to another device.
+  public func setAttrAccessible(_ attrAccessible: KeyAttrAccessible) -> Self {
```

Comment:
What will happen if I use a different value between instances? Do we need a migration?

Reply:
All data will remain accessible, the only difference would be that newer items would be able to migrate into other another devices, data stored with the previous `attrAccessible` won't migrate. A migration could be another path but it would require to read and save all the data again.

Reply:
Yes, let's keep without migration from our side and we can implement it later if needed